Classwork/techstart/javascript/hello_world.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+ <head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+   <title>Hello World v3.0</title>
+
+   <!-- Comments in HTML look like this -->
+   <!-- If the css file is in the same folder as your html,
+        and the file is name hello_world.css, you would write:
+        <link rel="stylesheet" href="hello_world.css"> -->
+   <link rel="stylesheet" href="hello_world.css">
+ </head>
+
+ <body>
+   <h1>Amazing Page Content</h1>
+
+   Hello world, we can include Javascript and CSS!
```

Comment:
Put this in a <p> tag
```suggestion
  <p> Hello world, we can include Javascript and CSS!</p>
```